thoth_user_api/api_v1.py

```diff
@@ -184,10 +184,11 @@ def list_runtime_environments(page: int=0):
     result = graph.runtime_environment_listing(page, PAGINATION_SIZE)
     return {
         "results": result,
-        "results_count": len(result),
+        "results_count": len(result)
```

Comment:
Results count is a meta-information so it should go to headers as well. Otherwise lgtm.